sdk/cosmos/azure-cosmos/azure/cosmos/_timeout_failover_retry_policy.py

```diff
@@ -75,24 +68,12 @@ def ShouldRetry(self, _exception):
 
         self.failover_retry_count += 1
 
-        if self.location_endpoint:
-            if _OperationType.IsReadOnlyOperation(self.request.operation_type):
-                # Mark current read endpoint as unavailable
-                self.global_endpoint_manager.mark_endpoint_unavailable_for_read(self.location_endpoint)
-            else:
-                self.global_endpoint_manager.mark_endpoint_unavailable_for_write(self.location_endpoint)
-
-        # set the refresh_needed flag to ensure that endpoint list is
-        # refreshed with new writable and readable locations
-        self.global_endpoint_manager.refresh_needed = True
```

Comment:
How will this change actually manifest to existing customers? Is there any conditions under which this might be breaking?